tests/neb/ui/test_get.py

```diff
@@ -126,3 +126,20 @@ def test_get_col_minor_version(neb_env, col_id, col_version, snapshot):
     ) as zip_dir:
         # THEN the complete zip is downloaded and matches the snapshot
         snapshot.assert_file_or_dir_match(zip_dir, snapshot_name)
+
+@markers.neb
+@markers.nondestructive
+@markers.parametrize("col_id,col_version", [("col11562", "1.19.6")])
+def test_get_col_with_resources(neb_env, col_id, col_version, snapshot):
+    # GIVEN neb, an environment name, a collection id, a collection version, and the snapshot tool
+    snapshot_name = get_neb_snapshot_name(col_id, col_version, "resources")
+
+    # WHEN we run `neb get --verbose env col_id col_version`
+    with Neb.get(
+        verbose=True, resources=True, env=neb_env, col_id=col_id, col_version=col_version, input="y"
+    ) as zip_dir:
+        # THEN the complete zip is downloaded and matches the snapshot
+        snapshot.assert_file_or_dir_match(zip_dir, snapshot_name)
```

Comment:
Does this need a [snapshot file](https://github.com/openstax/cnx-automation/tree/master/snapshots/tar_gz/neb/col11562) to be added for comparison?

I noticed `test_get_col_minor_version` doesn't have a corresponding `1.19.1.tar.gz` file (and instead the file gets written out through [this code path](https://github.com/openstax/cnx-automation/blob/f7ece379b32335ae90d05dd6d937713d83ffb268/fixtures/snapshot.py#L74)). I'm not sure if that's intentional for that test, but I suspect you may want to compare against a known snapshot for this one (?)